default/stringtables/en.txt

```diff
@@ -1928,7 +1928,7 @@ OPTIONS_DB_UI_SITREP_ICONSIZE
 Sets the sitrep icon width and height; default 16 (min 12, max 64).
 
 OPTIONS_DB_LOAD
-Loads the specified single-player save game.
+Loads and starts the specified single-player or multi-player save game. In a multi-player it requires hostless mode and no restrictions on a number of connected players because server will start without one.
```

Comment:
In multi-player, it requires hostless mode and no restriction on the minimum number of connected players, because the server will start with no connected players.

Reply:
Fixed.